vita3k/gui/src/manual.cpp

```diff
@@ -191,7 +191,7 @@ void draw_manual(GuiState &gui, EmuEnvState &emuenv) {
     // Draw left button
     if (current_page > 0) {
         ImGui::SetCursorPos(ImVec2(5.0f * SCALE.x, display_size.y - (40.0f * SCALE.y)));
-        if ((!hidden_button && ImGui::Button("<", BUTTON_SIZE)) || ImGui::IsKeyPressed(static_cast<ImGuiKey>(emuenv.cfg.keyboard_leftstick_left))) {
+        if ((!hidden_button && ImGui::Button("<", BUTTON_SIZE)) || ImGui::IsKeyPressed(static_cast<ImGuiKey>(emuenv.cfg.keyboard_leftstick_left)) || !max_scroll && wheel_counter == 1.f) {
```

Comment:
revert alkso this both, i have removed this for turn page for good reason, no corresponding how psvita works, and is complicated when manuel using scroll on page

Reply:
what do you mean "complicated when manuel using scroll on page"?
It is more comfortable to advance images with the mouse wheel.

Reply:
Is complicated like i said, lit manuel is like zoom, and using aleady scrolling for down/up page